### ISSUE TITLE
[Bug] Support for Accessibility automation properties on iOS is removed for Layouts starting from 4.0.0.346134-pre9

### ISSUE THREAD

**Issue by bijadi** [xamarin/Xamarin.Forms#7053]
### Description
Starting with 4.0.0.346134-pre9, accessibility no longer works on iOS for layouts
### Steps to Reproduce

1.  Create a simple layout page with automation properties as below.
```
<StackLayout
     AutomationId="Test"
     AutomationProperties.IsInAccessibleTree="True"
     AutomationProperties.Name="Test at layout">
     <Label 
       FontSize="18"
       Text="Test label" />
</StackLayout>
```
2.  Launch this page on iPhone.
3.  Enable accessibility voice over feature on iPhone
4. Note the missing support
### Expected Behavior
The stack layout in the above example has accessibility text read out. Also it should be visible in XCode Accessibility Inspector.
### Actual Behavior
The stack layout in the above example has accessibility text but it is not supported.
### Basic Information

- Version with issue: Starting with 4.0.0.346134-pre9 and above
- Last known good version: 4.0.0.304370-pre8
- IDE: Any
- Platform Target Frameworks: 
  - iOS:  12.10.0.157 
- Nuget Packages:4.0.0.346134-pre9 and above
- Affected Devices: All iOS devices

**Comment by slang25** [comment]
This issue might be related to https://github.com/xamarin/Xamarin.Forms/issues/6894

**Comment by jfversluis** [comment]
Confirmed that it worked (differently) in 4.0-pre8 than it does today.

[Repro7053.zip](https://github.com/xamarin/Xamarin.Forms/files/3467069/Repro7053.zip)
